### PR TITLE
Fix issues with OVRD and MEL from merge

### PR DESCRIPTION
**Pull request by blisteringherb**
#### What's this PR do?
This PR fixes the broken MEL index and incorrect fields on the OVRD recipient funnel indexes. These got broken from merging in this PR. https://github.com/DoSomething/quasar/pull/1018/files
#### What are the relevant tickets?
https://www.pivotaltracker.com/n/projects/2076969/stories/172679594

### CODE REVIEW

File: quasar/dbt/dbt_project.yml [hunk 1]
```diff
@@ -301,8 +301,8 @@ models:
           alias: ovrd_recipient_funnel
           materialized: table
           post-hook:
-            - "CREATE UNIQUE INDEX IF NOT EXISTS ovrd_r_index ON {{ this }}(northstar_id)"
-            - "CREATE INDEX ON {{ this }}(northstar_id)"
+            - "CREATE UNIQUE INDEX IF NOT EXISTS ovrd_r_index ON {{ this }}(user_id)"
```
Comment (sheyd) [inline quasar/dbt/dbt_project.yml:304]: I don't think we need to do `IF NOT EXISTS` in DBT index creation since we use `table` materialization mode?

Comment (blisteringherb) [reply quasar/dbt/dbt_project.yml:304]: Have we changed the outlook on this? Looks like we decided to add this clause back in August? https://github.com/DoSomething/quasar/pull/807#pullrequestreview-280891759

Comment (sheyd) [reply quasar/dbt/dbt_project.yml:304]: Hmmm, good point. But we have `CREATE INDEX` on other tables and we don't have runaway duplicate indexes. I'm guessing I was wrong and this was in relation to using Indexes when using `incremental` materialization mode. :thinking: It won't do any harm as is, so feel free to merge for expediency, but I'd like to figure out what the answer on this one is!

Comment (sheyd) [reply quasar/dbt/dbt_project.yml:304]: Approved so we can get this out.

Comment (blisteringherb) [reply quasar/dbt/dbt_project.yml:304]: On the upside, when and if we go to convert to incremental builds, we'll already have this code in place.

**Pull request merged**